core-lightning/hooks/pre-start

```diff
@@ -1,5 +1,33 @@
 #!/usr/bin/env bash
 
+# Ensure the ${APP_DATA_DIR}/app directory exists and is owned by the user (1000:1000).
+# This is required for app updates to new Blockstream app.
+
+set -euo pipefail
+
+APP_DATA_DIR="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/..)/data"
+
+[ ! -d "${APP_DATA_DIR}" ] && mkdir -p "${APP_DATA_DIR}" && chown 1000:1000 "${APP_DATA_DIR}"
+
+APP_SUBDIR="${APP_DATA_DIR}/app"
+
+[ ! -d "${APP_SUBDIR}" ] && mkdir -p "${APP_SUBDIR}" && chown 1000:1000 "${APP_SUBDIR}"
+
+# Create and set ownership for config.json file if it doesn't exist
+CONFIG_FILE_DESTINATION="${APP_SUBDIR}/config.json"
+CONFIG_FILE_CONTENTS='{
+  "unit": "SATS",
+  "fiatUnit": "USD",
+  "appMode": "DARK",
+  "isLoading": false,
+  "error": null
+}'
```

Comment:
Re duplication concern, we could technically remove the `data/app/config.json` file I think since this hook should always ensure it's created if it doesn't exist, for both fresh installs and updates.